styles/style.css

```diff
@@ -6,3 +6,128 @@ Green: #00B172
 White: #FFF
 
 */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700;800&display=swap');
+body {
+    font-family: 'Montserrat', sans-serif;
+    width: 80%;
+    margin: auto 10%;
+}
+
+/* header + image */
+#header-logo {
+    height: 50px;
+}
+
+.header-list {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.navbar {
+    position: fixed;
+    width: 80%;
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+    top: 0;
+    padding: 10px;
+    z-index: 1;
+}
+
+.header-list > li {
+    padding: 0 10px;
+    list-style-type: none;
+}
+
+.header-background {
+    background-image: url(/images/landing.jpg);
+    background-size: 100%;
+    background-repeat: no-repeat; 
+    height: 700px;
+    background-position: 50% 50%;
+    position: relative;
+    color: white;
```

Comment:
If you use _text-align: center_ here there is no need to use _position: absolute_ later.
For example: `.header-background {`
    `background-image: url('/images/landing.jpg');`
   ` background-size: 100%;`
    `background-repeat: no-repeat;`
   ` height: 700px;`
  `  position: relative;`
    `color: white;`
  `  text-align: center`
}

`.header-background>p {`
  `  font-size: 2em;`
   ` text-align: center;`
`}`

`.header-background h1 {`
    `padding-top: 200px;`
   ` font-size: 4em;`
`}`